docs/include/attributes-lsplugins.asciidoc

```diff
@@ -4,9 +4,10 @@ They correspond to the VPR settings in logstash-docs/docs/versioned-plugins/incl
 When we update one, we must update settings in the other location,
 
 Attribute text formatted without hard wrap is deliberate. 
-Otherwise, text breaks at return and content isn't displayed in its entirety. 
-/////
+Otherwise, text breaks at return and content after the return is dropped. 
 
+Text is written to accommodate multiple versions because plugins are not stack versioned.  
+/////
 
-:ecs-default: The `pipeline.ecs_compatibility` setting is available in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] and `pipelines.yml`. Any value in `pipeline.ecs_compatibility` other than `disabled` is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
 
+:ecs-default: When Logstash provides a pipeline.ecs_compatibility setting, its value is used as the default. Otherwise, the default value depends on which version of Logstash you are running: `disabled` for Logstash 7.x or `v8` for  Logstash 8.0 and later. You can opt out at the plugin, pipeline, or system level to maintain legacy behavior. For more information about ECS compatibility settings in Logstash and plugins, see {logstash-ref}/ls-ecs.
```

Comment:
:thinking:

I am very wary of what I call "false clarity" creeping in here. I want our users to understand that there are levels of granularity to opting-in or -out of ECS compatibility so that they can move forward with the right amount of caution and without fear.

Assuming that this text is referenced in each plugin's section for its`ecs_compatibility` option:

```suggestion
:ecs-default: When the `ecs_compatibility` option for this plugin is not explicitly given, its effective value depends on the `pipeline.ecs_compatibility` setting for the pipeline in `pipelines.yml`, or globally in {logstash-ref}/logstash-settings-file.html[`logstash.yml`], allowing you to specify your preferred behavior at the plugin, pipeline, or system level. If no preference is specified, the default value is `v8` for Logstash 8 or `disabled` for all earlier releases of Logstash. For more information about ECS compatibility settings in Logstash and plugins, see {logstash-ref}/ls-ecs.
```

***

Note: We introduced the `pipeline.ecs_compatibility` setting in Logstash 7.10. Any plugin running on an older Logstash that didn't specify `ecs_compatibility` for the individual plugin instance wouldn't be able to _configure_ it for the pipeline or globally without that setting but the effective value would be `disabled` (just as if the setting were present but also left unspecified).

Reply:
> Assuming that this text is referenced in each plugin's section for itsecs_compatibility option
That's the plan. Glad we're on the same page. :-)